lesson2/homework 3(x^y).py

```diff
@@ -0,0 +1,9 @@
+if __name__ == '__main__':
+    x=25
+    n=5
+    i=1
+    num=1
+    while i<=n:
```

Comment:
You should use for cycle, because number of iterations is defined.

Reply:
Done